tools/compression/mkcompresstool.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
This script must be called automatically during build time to compile the compression algorithm. 
Please check mksyscall in os/tools folder and make this similar to that. The source file as well as the executable for the compression algorithm must be in os/tools.

Reply:
will make this change

Reply:
made change in Makefile.unix for automatic build of compression tool.
Please review and suggest an appropriate place to call rule for compression tool creation.